model/infinispan/src/main/java/org/keycloak/models/sessions/infinispan/entities/AuthenticatedClientSessionEntity.java

```diff
@@ -83,6 +85,28 @@ public void setTimestamp(int timestamp) {
         this.timestamp = timestamp;
     }
 
+    public int getUserSessionStarted() {
+        String started = getNotes().get(AuthenticatedClientSessionModel.USER_SESSION_STARTED_AT_NOTE);
+        return started == null ? timestamp : Integer.parseInt(started);
+    }
+
+    public int getStarted() {
```

Comment:
This defaults to `timestamp` while `AuthenticatedClientSessionModel.getStarted` defaults to 0 if `STARTED_AT_NOTE` is not set. Is it intentional?

Reply:
Yep, the same.